src/services/api.ts

```diff
@@ -110,12 +126,14 @@ export async function getFee({
   kind,
   amount,
   network,
+  endpoint,
```

Comment:
the more instance variables we are passing down in these free methods (network, endpoint), the more I feel like maybe we shouldn't expose the free methods and require consumers to user our instance. But up to you if you find them useful.

Reply:
Makes sense. So far these functions are used only for the withdraw script, I'll change it later by removing the export when I change the withdraw script.